tsconfig.json

```diff
@@ -0,0 +1,67 @@
+{
+    "compilerOptions": {
+      /* Basic Options */
+      "target": "ES5", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
+      "module": "commonjs", /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
```

Comment:
Can we get rid of the comments and commented out config options?

Reply:
done.